Blur.txt

```diff
@@ -0,0 +1,9 @@
+Name:Blur
+ManaCost:2 U
+Types:Instant
+A:SP$ ChangeZone | Cost$ 2 U | ValidTgts$ Creature.YouCtrl | TargetMin$ 1 | TargetMax$ 1 | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select target creature you control | RememberTargets$ True | SubAbility$ DBReturn | SpellDescription$ Exile target creature you control, then return that card to the battlefield under their owner's control.
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ All | Destination$ Battlefield | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | NumCards$ 1 | SpellDescription$ Draw a card.
+AI:RemoveDeck:All

```

Comment:
Hmm, seems like the AI could handle this... why did you add this? Is there another blink type spell with this tag?

Reply:
I cannot remember which card I referenced that had it. Looking at "Acrobatic Maneuver" (which is the same card but white) it does not have that line so I'll just remove it.